pkg/symbol/elfutils/elfutils.go

```diff
@@ -25,6 +25,32 @@ import (
 	"github.com/nanmu42/limitio"
 )
 
+// File represents an open ELF file.
+type File struct {
```

Comment:
Can we get away without introducing a new type? It'd be amazing just to use `elf.File`?

Reply:
Sure, I've updated the PR.